src/parquet/parquet_reader_writer-test.cc

```diff
@@ -0,0 +1,330 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
My motivation is to add end to end writer-reader tests for Parquet which we currently don't have.